src/components/dashboard/AnimatedSwipeableFlatList.js

```diff
@@ -0,0 +1,4 @@
+import { SwipeableFlatList } from 'react-native-swipeable-lists'
```

Comment:
This is because an issue when reloading createAnimatedComponent

Reply:
please add comments inside code explaining hacks and fixes